node/src/commands/deploy.js

```diff
@@ -2,41 +2,41 @@ const DeployUtils = require('../lib/deploy-utils');
 const logger = require('../lib/logger');
 const { options } = require('../config/constants');
 
-const { BLUEPRINT_ID } = options;
-
-const setConfigurationFromOptions = async (environmentVariables, environment, blueprintId) => {
-  const deployUtils = new DeployUtils();
-
-  if (environmentVariables && environmentVariables.length > 0) {
-    for (const config of environmentVariables) {
-      logger.info(
-        `Setting Environment Variable ${config.name} to be ${config.value} in environmentId: ${environment.id}`
-      );
-      await deployUtils.setConfiguration(environment, blueprintId, config.name, config.value, config.sensitive);
-    }
-  }
-};
+const { BLUEPRINT_ID, REVISION, ENVIRONMENT_NAME, PROJECT_ID } = options;
 
 const assertBlueprintExistsOnInitialDeployment = options => {
   if (!options[BLUEPRINT_ID]) throw new Error('Missing blueprint ID on initial deployment');
+  if (!options[REVISION]) throw new Error('Missing revision on initial deployment');
```

Comment:
Why?
We don't have to have a revision, every git repo has a default branch.

Reply:
I looked only on `create` where it gets it from request but then on `deploy` it's not required 🤦 
64f9e72